scripts/apply_extra

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+EXTRA="/app/extra"
+TARBALL="$EXTRA/AlderonGamesLauncher-1.3.6-linux-x64.tar.xz"
```

Comment:
so you need to change this at each release?

Reply:
This was an oversight on my part, thanks for bringing it up